src/index.js

```diff
@@ -71,7 +71,7 @@ function main(client, options, _performance) {
 			}
 		});
 
-		all(promises).then(function () {
+        promises.length && all(promises).then(function () {
```

Comment:
Whats the reasoning behind doing this check instead of checking the contaminations list and not confirming for any candidate that has been contaminated?